locales/en-US.json

```
@@ -8,6 +8,7 @@
   "login": "Login",
   "logout": "Logout",
   "more": "More",
+  "moreLikeThis": "More Like This",
```

Pretty much all of the rest is written in sentence case, which I feel looks cleaner. It's probably best to respect one style, to be consistent.
```suggestion
  "moreLikeThis": "More like this",
```